src/types.ts

```diff
@@ -65,10 +65,17 @@ export type OAuthConfig = {
   secret: string;
 };
 
-type MetricsConfig = {
-  host: string;
-  port: number;
-  prefix?: string;
+export type EventData = {
+  landRequestId?: string;
+  pullRequestId?: string;
+  targetBranch?: string;
+  commit?: string;
+  duration?: number;
+};
```

Comment:
Nice, maybe `sourceBranch` would be worthwhile also in addition to `targetBranch`